app/controllers/new_admin_controller.rb

```diff
@@ -0,0 +1,11 @@
+class NewAdminController < ApplicationController
+  include NewAdmin::Pagination
+
+  before_action :authenticate_user!
```

Comment:
your are missing the `load_and_authorize_resource` here

Reply:
If I put `load_and_authorize_resource` in `NewAdminController`, controllers without a model will break:

For example:
![image](https://github.com/Codeminer42/Punchclock/assets/30778707/15114cb6-866b-4881-a295-f604516129e7)

Reply:
So this needs to be explicitly added for all new admin controllers